pact/kpenny/kpenny.repl

```diff
@@ -0,0 +1,326 @@
+
```

Comment:
let's add this to CI https://github.com/kadena-io/kadenaswap/actions/runs/531551264/workflow#L36-L47